spec/factories.rb

```diff
@@ -246,4 +246,36 @@
     repository
     html_body { "Welcome to the jungle" }
   end
+
+  factory :raw_upstream_data, class: RepositoryHost::RawUpstreamData do
+    archived { true }
+    default_branch { "main" }
+    description { "description" }
+    fork { true }
+    fork_policy { "" }
+    forks_count { 10 }
+    full_name { "test/repo" }
```

Comment:
🎱 Would be nice to sequence this / some other traits so `create_list` can create non-identical-except-for-id objects.